src/shared/components/palette/case-flag/components/manage-case-flags/manage-case-flags.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="form-group" [formGroup]="formGroup">
+  <div class="govuk-form-group" [ngClass]="{'form-group-error': errorMessages.length > 0}">
+    <fieldset class="govuk-fieldset" aria-describedby="flag-location-heading">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+        <h1 id="flag-location-heading" class="govuk-fieldset__heading">
+          {{flagLocationTitle}}
```

Comment:
@tarikdogan-hmcts - this ought to be `{{flagSelectionTitle}}` (will also need changing in the component class)